### DISCUSSION TITLE
media queries are not working

### DISCUSSION THREAD

**Discussion by abyii** [tailwindlabs/tailwindcss#11230] [Help] [answered]
# Media queries are not working.

### Problem:
Media queries are not working

### Setup:
I have a turborepo monorepo with pnpm and following dependencies:
 - next js: "13.3.0"
 - postcss: "^8.4.23"
 - tailwindcss: "^3.3.2"
 - autoprefixer: "^10.4.14"

This is my postcss.config.js:

```js
module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
  },
};
```

### What I found so far:

the classes are getting added but they are being overridden by regular classes, because the regular classes appear after the md:* classes in the css file. I'm not sure if this is a problem with tailwindcss or postcss. 

<img width="234" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/75441358/f61fbc98-0c11-4499-ab3c-02ec367eaf53">

### Expected behavior:
Media queries should work. 

Please help ;-;
Thankyou.

**Comment by wongjn** [comment]
It seems like `flex-col` is being added twice to your `layout.css` file. This could indicate you have multiple instances of Tailwind compiling, whereas you'd only want to rely on Tailwind running once. For example, multiple instances could be running if you have separate CSS files that you compile with Tailwind and then they are concatenated together. Alternatively, it could be that one `flex-col` is from your Tailwind compilation run, and the other comes from a third-party library's CSS. This isn't an exhaustive list but it should hopefully give you some ideas on where to troubleshoot.

**Reply by abyii** [reply, marked as answer]
Thanks a lot! that was exactly the issue. I had an internal ui library that was also using tailwind and the web app was importing the built stylesheet from that ui library and the classes were clashing.

I fixed it by adding a prefix to ui library tailwind classes and making sure that i import web app's tailwind stylesheet _before_ the ui library's stylesheet. 

```js
// ui/tailwind.config.js
const sharedConfig = require("tailwind-config/tailwind.config.js");

module.exports = {
  prefix: "um-",
  darkMode: ["class", '[data-theme="dark"]'],
  ...sharedConfig,
};
```

```js
// webapp 
import "../styles/globals.css";
import "ui/styles.css";
```

Thanks again :)

**Reply by brianwachira** [reply]
Hi there, I have imported tailwind global config in one of my apps like this.
![image](https://github.com/tailwindlabs/tailwindcss/assets/28436092/5b381f69-c38e-4754-9fd6-18620e754d62)

I keep getting the error below 
![image](https://github.com/tailwindlabs/tailwindcss/assets/28436092/32c83180-feb8-4955-9f2e-32d1847a4803)

Here is my app's package.json
![image](https://github.com/tailwindlabs/tailwindcss/assets/28436092/73744df6-31fa-4711-a2d6-358ae93f643e)

Here is tailwind-config package.json
![image](https://github.com/tailwindlabs/tailwindcss/assets/28436092/7383e769-1c09-492c-9ae0-e5f1eea59df1)


What could be the issue?
